### PR TITLE
Adds Armolitskiy's and Eluxor's sprites and items at Reere's Request

### PR DESCRIPTION
**Pull request by Trexdude**
## About The Pull Request

Adds different sprites for and a cape to TE's hardsuit, adds a jacket for the TE, adds Technomancer armor, adds a bomj coat to random loot and loadout, and updates the sprites of the Technomancer and TE jumpsuits. Hardsuits now show modules on the person wearing them if the modules are set to, may cause unforseen consequences far as I know. Sprites by Armolitskiy (Armolitskiy#4866 on Discord) and Eluxor (Eluxor#8938 on Discord), requested by Reere.

## Why It's Good For The Game

Requested by Reere.

## Changelog
:cl:
add: TE's hardsuit now has a cape module, and TE has a jacket
add: TM's now have access to heat and energy weapon-resistant armor, and helmets
add: Bomj coat added to loot and loadout
imageadd: Added new TE and TM jumpsuit sprites from Armolitskiy and Eluxor (req. by Reere)
imagedel: Removed old TE and TM jumpsuit icons
code: Hardsuit modules now can appear on your sprite if module is set to
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -311,6 +311,29 @@
...
+/obj/item/clothing/suit/storage/vest/insulated
+	name = "insulated armor"
+	desc = "A set of armor insulated against heat and electrical shocks, shielded against radiation, and protected against energy weapon projectiles."
+	icon_state = "armor_engineering"
+	item_state = "armor_engineering"
+	blood_overlay_type = "armor"
+	armor = list(
+		melee = 30,
+		bullet = 25,
+		energy = 70,
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/armor.dm:324]: Lmao this is too much, make it 40 at least.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -139,6 +139,16 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	siemens_coefficient = 2
 
+/obj/item/clothing/suit/bomj
+	name = "bomj coat"
+	desc = "A coat padded with synthetic insulation."
+	icon_state = "bomj"
+	item_state = "bomj"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/miscellaneous.dm:147]: Should cover legs.

File: code/modules/clothing/suits/jobs.dm [hunk 3]
```diff
@@ -165,6 +165,24 @@
...
 	price_tag = 50
 
+//Chief Engineer/Technomancer Exultant
+/obj/item/clothing/suit/storage/te_coat
+	name = "exultant coat"
+	desc = "A sturdy and proud crimson coat. Lightly armored, with some protection against radiation."
+	icon_state = "te_coat"
+	item_state = "te_coat"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/jobs.dm:175]: Should cover legs.

**Comment by SpadesSlick** [comment]
image
This is clearly wrong.

**Comment by SpadesSlick** [comment]
image
And those as well.

File: code/modules/clothing/suits/jobs.dm (hunk 3 above)
Comment (Trexdude) [reply code/modules/clothing/suits/jobs.dm:175]: The sprite doesn't cover the legs, but aye.

Comment (Trexdude) [reply code/modules/clothing/suits/jobs.dm:175]: Unless this is for the heat resistance, fair nough then far as I know.

Comment (SpadesSlick) [reply code/modules/clothing/suits/jobs.dm]: It's long enough to do that.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -139,6 +139,16 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	siemens_coefficient = 2
 
+/obj/item/clothing/suit/bomj
+	name = "bomj coat"
+	desc = "A coat padded with synthetic insulation."
+	icon_state = "bomj"
+	item_state = "bomj"
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/miscellaneous.dm:146]: It needs at least some armor protection. On a lever of other coats.

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 5]
```diff
@@ -445,3 +445,13 @@
...
+
+/obj/item/rig_module/cape
+	name = "cape"
+	desc = "A cape designed to be attached to hardsuits."
+
+/obj/item/rig_module/cape/te
+	name = "technomancer cape"
+	desc = "A tough regal cape, imprinted with the emblem of the Technomancer League."
+	suit_overlay_active = "cape_te"
+	suit_overlay_inactive = "cape_te"
```
Comment (SpadesSlick) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:457]: Please add endline.

File: code/modules/clothing/suits/miscellaneous.dm (hunk 4 above)
Comment (Trexdude) [reply code/modules/clothing/suits/miscellaneous.dm:146]: Any specific values?

Comment (SpadesSlick) [reply code/modules/clothing/suits/miscellaneous.dm:146]: Literally just copypaste some coat.

**Pull request merged**